plugins/module_utils/fusion.py

```diff
@@ -113,6 +114,9 @@ def get_fusion(module):
         try:
             config.issuer_id = issuer_id
             config.private_key_file = key_file
+            if PARAM_PRIVATE_KEY_PASSWORD in module.params:
+                config.private_key_password = module.params[PARAM_PRIVATE_KEY_PASSWORD]
```

Comment:
Oops, I probably didn't publish my comment on this. Please, fix this. If parameter is not present, it is still in dictionary but with value `None`.

Plase, do as with other parameters already present in this module.
```suggestion
            private_key_password = module.params[PARAM_PRIVATE_KEY_PASSWORD]
            if private_key_password is not None:
                config.private_key_password = private_key_password
```

Reply:
agree, initially I tried to do like that, but for some reason it failed to work. fixed now